src/test/java/integration/JUnitTestsTest.java

```diff
@@ -357,6 +371,18 @@ void forgotPropertyAnnotation() {
             assertThat(result.getTests().hasTestsFailingOrFailures()).isFalse();
         }
 
+        // Multiple class names contain the substring "Test" -> test discovery fails
+        @Test
+        void multipleTestClassesDiscovered() {
```

Comment:
Maybe the name should say something about that "it should fail"

Reply:
Sure, I renamed it to `multipleTestClassesDiscoveredShouldThrowException`